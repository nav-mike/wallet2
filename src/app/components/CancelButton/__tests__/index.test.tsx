import * as React from 'react';
import { render } from '@testing-library/react';
import { CancelButton } from '..';

describe('<CancelButton />', () => {
  it('should be render and match the snapshot', () => {
    const button = render(<CancelButton />);
    expect(button.container.firstChild).toMatchSnapshot();
  });
});
