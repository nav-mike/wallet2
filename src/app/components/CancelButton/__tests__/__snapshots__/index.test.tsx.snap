// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CancelButton /> should be render and match the snapshot 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: rgba(220,120,95,1);
  text-align: center;
}

.c0:hover {
  cursor: pointer;
  background-color: #fff5f2;
  border: 1px solid #fff5f2;
  border-radius: 0.5rem;
  min-height: 2rem;
}

<a
  class="c0"
/>
`;
